docs/RELEASING.md

```diff
@@ -8,6 +8,15 @@ Major and minor versions follow the versioning in the Echoes Design System. Patc
 
 For more details, refer to the [internal versioning strategy](https://docs.google.com/document/d/1JkzJVFBN3MPl-uBDgddICfdFf4kXPaRBa6XqaJV0F5U) (Sonar employees only)
 
+## Release Process for Release Candidates
```

Comment:
We also need to make sure to bump the version in the package.json with the correct version-rcx format before doing the github release right?